src/account/AbstractKeylessAccount.ts

```diff
@@ -203,6 +232,27 @@ export abstract class AbstractKeylessAccount extends Serializable implements Key
       throw new Error("Cannot serialize - proof undefined");
     }
     this.proof.serialize(serializer);
+    serializer.serializeOption(this.verificationKeyHash, 32);
+  }
+
+  static partialDeserialize(deserializer: Deserializer): {
```

Comment:
cannot - deserialize namespace becomes too crowded.  then when KeylessAccount implements deserialize theres an error because return type doesn't match